packages/rx_bloc_cli/mason_templates/bricks/lib_auth_apple/README.md

```diff
@@ -0,0 +1,25 @@
+# lib_auth_apple
+
+[![Powered by Mason](https://img.shields.io/endpoint?url=https%3A%2F%2Ftinyurl.com%2Fmason-badge)](https://github.com/felangel/mason)
+
+A brick created to add social log in with Apple functionality to our application. Use third party package, so in order to make it work, fulfill requirements, described into its [documentation][5]. 
```

Comment:
This description has a few grammatical/spelling mistakes

Reply:
Edited